normandy/recipes/models.py

```diff
@@ -127,6 +128,12 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs):
+        if self.revision_id:
+            self.revision_id += 1
```

Comment:
This will result in nothing ever having a positive `revision_id`, right? Since the default is 0? That doesn't seem right.